style.css

```diff
@@ -0,0 +1,844 @@
+.my-project {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto;
+  gap: 10px;
+  padding: 10px;
+}
+
+h1 {
+  position: absolute;
+  width: 181px;
+  height: 50px;
+  left: 16px;
+  top: 20px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.project1 {
+  position: absolute;
+  width: 100%;
+  height: 210px;
+  left: 0;
+  top: 120px;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+}
+
+.works {
+  position: absolute;
+  width: 100%;
+  height: 156px;
+  left: 0;
+  top: 329px;
+  background: #1a2236;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(0.5px);
+  display: flex;
+  flex-direction: column;
+}
+
+h2 {
+  position: absolute;
+  width: 264px;
+  height: 16px;
+  left: 20px;
+  top: 1px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 24px;
+  line-height: 80%;
+  color: #fff;
+}
+
+.languages {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: baseline;
+}
+
+.mark-up {
+  background: #98dede;
+  transition: all 0.5s;
+  cursor: pointer;
+  position: absolute;
+  width: 80px;
+  height: 20px;
+  font-size: 12.5px;
+  left: 15px;
+  top: 60px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.react {
+  background: #98dede;
+  transition: all 0.5s;
+  cursor: pointer;
+  position: absolute;
+  width: 95px;
+  height: 20px;
+  right: 25px;
+  left: 105px;
+  top: 60px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 12.5px;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.responsive {
+  background: #98dede;
+  transition: all 0.5s;
+  cursor: pointer;
+  position: absolute;
+  width: 70px;
+  height: 20px;
+  font-size: 12.5px;
+  left: 210px;
+  top: 60px;
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.show-works {
+  font-family: sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  text-align: center;
+  color: #fff;
+  position: absolute;
+  height: 50px;
+  width: 210px;
+  left: 15px;
+  top: 90px;
+  background: #1a2236;
+  border: 1px solid #fff;
+  backdrop-filter: blur(0.5px);
+  transition: all 0.5s;
+  cursor: pointer;
+}
```

Comment:
As requested by the project requirements:
![image](https://user-images.githubusercontent.com/42799579/198790278-3e2bd8cb-a281-4d6f-9baa-1672124ad08c.png)

Kindly add the corresponding button effects to your buttons: 
![image](https://user-images.githubusercontent.com/42799579/198786838-917e1b18-560c-4a2c-ae94-cb3888b6e537.png)
You should implement the hover & pressed effects 😃.